docs/user-guide/test-zen-doc_Sean.md

```diff
@@ -53,50 +52,72 @@ FTP Port  |The FTP Port number for internal use. The default port is 21. If not
 User Name |Your z/OS username.
 Password  |Your z/OS password.
 
-        1. Select Use FTP over TLS (Highly Recommended). Provide details as described in the following table:
+3. Select Use FTP over TLS (Highly Recommended). Provide details as described in the following table:
         
 Field name | Description
 ---|---
 Min TLS    |Select the minimum TLS version to accept the certificate from the server.
 Max TLS    |Select the maximum TLS version to accept the certificate from the server.
-        2. Select **Accept all certificates**.
-3. Click **VALIDATE CREDENTIALS**.
-4. Click **Save & Close** if you want to go back to the **New Zowe Installation** screen.
-5. Click **Continue**.
-6. In the **Before you start** window, review the instructions. 
-7. In the **Job statement** field, customize the job statement if needed and click **SAVE AND VALIDATE**.
-8. In the **Planning window**, provide details for z/OS Unix locations, identifiers, and z/OSMF details. 
+4. Select **Accept all certificates**.
+5. Click **VALIDATE CREDENTIALS**.
+6. Click **Save & Close** if you want to go back to the **New Zowe Installation** screen.
```

Comment:
Maybe we can have this note somewhere separately mentioning that we can save the installation progress, like below together with other potential actions - View/Edit YAML / View/Submit Job / Skip a step. 
Or remove this line, as it should be obvious form the UI